.rubocop_todo.yml

```diff
@@ -343,6 +343,7 @@ Metrics/ModuleLength:
     - 'spec/services/permissions/order_spec.rb'
     - 'spec/services/variant_units/option_value_namer_spec.rb'
     - 'spec/support/request/stripe_stubs.rb'
+    - 'spec/lib/stripe/payment_intent_validator_spec.rb'
```

Comment:
The problem here is that the spec is within the `Stripe` module. You can remove the module and add `Stripe::` where needed, for example the top `describe` block.

Reply:
Ah, nice - I did just that on beb47f03c02640f829a41e4cb94ddd0fd747a201.
Thank you.